chromium_src/chrome/browser/ui/views/frame/browser_view.h

```diff
@@ -31,11 +31,18 @@
 
 #if BUILDFLAG(IS_WIN)
 #define GetSupportsTitle virtual GetSupportsTitle
+
+// On Windows <winuser.h> defines LoadAccelerators
+#pragma push_macro("LoadAccelerators")
```

Comment:
@goodov is this kosher?

Reply:
FWIW, it looks like Chromium just undefines it:
`chrome/browser/ui/views/frame/browser_view.cc#298`